From a9596c7492e0ac9dd74b5e921abdf8a36624fbf5 Mon Sep 17 00:00:00 2001
From: Chen Hanxiao <chen_han_xiao@126.com>
Date: Tue, 13 Feb 2018 16:40:56 +0800
Subject: [PATCH 29/45] pylint: fix logging-too-few-args issue

From: Chen Hanxiao <chenhanxiao@gmail.com>

pylint complain:
  Not enough arguments for logging format string (logging-too-few-args)

Signed-off-by: Chen Hanxiao <chenhanxiao@gmail.com>
---
 tests/uitests/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/uitests/utils.py b/tests/uitests/utils.py
index e481e443..1f7d61e8 100644
--- a/tests/uitests/utils.py
+++ b/tests/uitests/utils.py
@@ -176,7 +176,7 @@ class _FuzzyPredicate(dogtail.predicate.Predicate):
         except Exception as e:
             logging.debug(
                     "got predicate exception name=%s role=%s labeller=%s: %s",
-                    (self._name, self._roleName, self._labeller_text, e))
+                    self._name, self._roleName, self._labeller_text, e)
 
 
 def check_in_loop(func, timeout=2):
-- 
2.16.2

